### PR TITLE
Nv 311 switch environment

### PR DESCRIPTION
**Pull request by ninjadev101**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
#482

### CODE REVIEW

File: apps/web/src/components/layout/components/SideNav.tsx [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+import { useState, useContext } from 'react';
+import { useQuery, useQueryClient } from 'react-query';
 import { Navbar } from '@mantine/core';
+import { IEnvironment } from '@novu/shared';
+import { getMyEnvironments, getCurrentEnvironment } from '../../../api/environment';
+import { api } from '../../../api/api.client';
 import { NavMenu, SegmentedControl } from '../../../design-system';
 import { Activity, Bolt, Box, Settings, Team } from '../../../design-system/icons';
+import { AuthContext } from '../../../store/authContext';
+import { act } from 'react-dom/test-utils';
```
Comment (davidsoderberg) [inline apps/web/src/components/layout/components/SideNav.tsx:10]: Is this a mistake or why is this imported here?

Comment (ninjadev101) [reply apps/web/src/components/layout/components/SideNav.tsx:10]: Idk.. did you add it? Lol

**Review by davidsoderberg — APPROVED** [review]
Great work, looks good 👍 
Thanks for all contributions ⭐

File: apps/web/src/components/layout/components/SideNav.tsx (hunk 1 above)
Comment (davidsoderberg) [reply apps/web/src/components/layout/components/SideNav.tsx]: haha now :P

**Pull request merged**